docs/classad-attributes/scheduler-classad-attributes.rst

```diff
@@ -432,238 +316,158 @@ Scheduler ClassAd Attributes
     have exited in the previous time interval defined by attribute
     ``RecentStatsLifetime`` had spent waiting to start.
 
-:index:`RecentJobsBadputRuntimes<single: RecentJobsBadputRuntimes; ClassAd Scheduler attribute>`
-
-``RecentJobsBadputRuntimes``:
+:classad-attribute:`RecentJobsBadputRuntimes`
     A Statistics attribute defining a histogram count of jobs that did
     not complete successfully, as classified by time spent running, in
     the previous time interval defined by attribute
     ``RecentStatsLifetime``. Counts within the histogram are separated
     by a comma and a space, where the time interval classification is
     defined in the ClassAd attribute ``JobsRuntimesHistogramBuckets``.
 
-:index:`RecentJobsBadputSizes<single: RecentJobsBadputSizes; ClassAd Scheduler attribute>`
-
-``RecentJobsBadputSizes``:
-    A Statistics attribute defining a histogram count of jobs that did
-    not complete successfully, as classified by image size, in the
-    previous time interval defined by attribute ``RecentStatsLifetime``.
+:classad-attribute:`RecentJobsBadputSizes`
     Counts within the histogram are separated by a comma and a space,
     where the size classification is defined in the ClassAd attribute
     ``JobsSizesHistogramBuckets``.
 
-:index:`RecentJobsCheckpointed<single: RecentJobsCheckpointed; ClassAd Scheduler attribute>`
-
-``RecentJobsCheckpointed``:
+:classad-attribute:`RecentJobsCheckpointed`
     A Statistics attribute defining the number of times jobs that have
     exited with a *condor_shadow* exit code of ``JOB_CKPTED`` in the
     previous time interval defined by attribute ``RecentStatsLifetime``.
 
-:index:`RecentJobsCompleted<single: RecentJobsCompleted; ClassAd Scheduler attribute>`
+:classad-attribute:`RecentJobsCompleted`.
 
-``RecentJobsCompleted``:
-    A Statistics attribute defining the number of jobs successfully
-    completed in the previous time interval defined by attribute
-    ``RecentStatsLifetime``.
-
-:index:`RecentJobsCompletedRuntimes<single: RecentJobsCompletedRuntimes; ClassAd Scheduler attribute>`
-
-``RecentJobsCompletedRuntimes``:
-    A Statistics attribute defining a histogram count of jobs that
-    completed successfully, as classified by time spent running, in the
-    previous time interval defined by attribute ``RecentStatsLifetime``.
+:classad-attribute:`RecentJobsCompletedRuntimes`.
     Counts within the histogram are separated by a comma and a space,
     where the time interval classification is defined in the ClassAd
     attribute ``JobsRuntimesHistogramBuckets``.
 
-:index:`RecentJobsCompletedSizes<single: RecentJobsCompletedSizes; ClassAd Scheduler attribute>`
-
-``RecentJobsCompletedSizes``:
-    A Statistics attribute defining a histogram count of jobs that
-    completed successfully, as classified by image size, in the previous
-    time interval defined by attribute ``RecentStatsLifetime``. Counts
+:classad-attribute:`RecentJobsCompletedSizes`. Counts
```

Comment:
Ditto.  Old lines 535 to 538 should remain, and new line 379 go.  Then change old line 535 to use classad-attribute, I gues.s